src/extension.ts

```diff
@@ -127,6 +125,44 @@ function initAsyncCommand(context: vscode.ExtensionContext, commandId: string, c
 	}));
 }
 
+async function update(): Promise<void> {
+	if (lastOpenedDocumentType === DocumentType.Mongo) {
+		await MongoCommands.updateDocuments(connectedDb, lastCommand, lastOpenedMongoDocument);
+	}
+	else if (lastOpenedDocumentType === DocumentType.DocDB) {
+		await CosmosDBCommands.updateDocDBDocument(lastOpenedDocDBDocument);
+	}
+}
+
+async function updateOpenDocumentIfChanged(): Promise<void> {
+	let oldDocument, newDocument, recentDoc;
```

Comment:
I don't like declaring variables like this. If you move everything into the try block then you can see what the variable is and declare it inline:

```
const oldDocument = recentDoc.data;
```